src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -9652,796 +9666,795 @@ namespace AirflowNetworkBalanceManager {
         int S2;
         int R1;
         int R2;
-        static bool OneTimeFlag(true);
-        static bool ErrorsFound(false);
         bool LocalError;
         Array1D_bool NodeFound;
-        Real64 FanFlow;
-        static bool IsNotOK(false);
-        static bool errFlag(false);
+
+        bool ErrorsFound(false);
+        bool IsNotOK(false);
+        bool errFlag(false);
```

Comment:
Making these static is great.  There was no need for ErrorsFound, IsNotOK, or errFlag to be static in the first place.  Having a static OneTimeFlag in the function is the core issue with this function, and I am glad to see it is gone.